src/satellite_consumer/run.py

```diff
@@ -23,11 +26,29 @@
 from satellite_consumer.storage import create_empty_zarr, create_latest_zip, get_fs, write_to_zarr
 from satellite_consumer.validate import validate
 
+# Load environment variables from .env file
+load_dotenv()
+
+# Get Sentry DSN from environment variables
+SENTRY_DSN = os.getenv("SENTRY_DSN")
+
+# Initialize Sentry if DSN is available
+if SENTRY_DSN:
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,  # Using .env variable
+        traces_sample_rate=1.0,
+        environment="production"
```

Comment:
Can this value be pulled from an environment variable called ENVIRONMENT instead of hard coded?